### DISCUSSION TITLE
Push latest version to Go?

### DISCUSSION THREAD

**Discussion by jenn-mitchell** [AlistGo/alist#817] [Q&A]
Will the latest release be published to Go? The latest version available on Go is 1.0.6.

Thanks!

**Comment by xhofe** [comment]
The crawling of `go.dev` seems to be an automatic process and cannot be actively pushed, and the reason for not grabbing the new version should be because the name of the main branch is not `main` or `master`.
Additional, this project is an app not a library, I don't think it's necessary to push to `go.dev`
